docs/_static/css/custom-theme.css

```diff
@@ -18,4 +80,155 @@
 
 .wy-nav-content {
     border-left: 2px #EEEEF1 solid;
-}
\ No newline at end of file
+    max-width: 1200px;
+}
+.caption {
+    border-top: 1px solid #eeeef1;
+}
+.caption-text {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:500;
+    font-size: 12px;
+    color: #666669;
+    line-height: 14px;
+    text-transform: capitalize;
+}
+
+.current {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:400;
+    font-size: 13px;
+    color: #555558;
+    line-height: 13px;
+    background: #fff;
+}
+
+.reference.internal {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:400;
+    font-size: 13px;
+    color: #555558;
+    line-height: 20px;
+    text-decoration: none;
+}
+.wy-menu-vertical a:hover {
```

Comment:
`#d9d9d9` for `a:hover span.toctree-expand` seems a bit too light to me

Reply:
i think that's fine. users will hover and click on the texts to drill down the content.